clldutils/text.py

```diff
@@ -15,8 +15,7 @@ def strip_brackets(text, brackets=None):
     stack = []
     tmp_sequence = list(text)
     new_sequence = ''
-    while tmp_sequence:
-        itm = tmp_sequence.pop(0)
+    for itm in tmp_sequence:
```

Comment:
you don't even have to cast to `list` because `str` already implements `__iter__`. So
```python
for itm in text:
```
should do.